tests/ui/typeck/duplicate_overloaded_items.sol

```diff
@@ -12,32 +12,32 @@ event E3(uint) anonymous;
 event E4(uint); //~ ERROR: event with same name and parameter types declared twice
 event E4(uint indexed);
 
-function f1() public {} //~ ERROR: function with same name and parameter types declared twice
-function f1() public {}
+function f1() {} //~ ERROR: function with same name and parameter types declared twice
+function f1() {}
 
-function f2() public {} //~ ERROR: function with same name and parameter types declared twice
-function f2() public {}
-function f2() public {}
+function f2() {} //~ ERROR: function with same name and parameter types declared twice
+function f2() {}
+function f2() {}
 
-function f2_2() public {}
-function f2_2(int) public {}
-function f2_2(uint) public {} //~ ERROR: function with same name and parameter types declared twice
-function f2_2(uint) public {}
+function f2_2() {}
+function f2_2(int) {}
+function f2_2(uint) {} //~ ERROR: function with same name and parameter types declared twice
+function f2_2(uint) {}
 
-function f3(int) public {}
-function f3(uint) public {}
+function f3(int) {}
+function f3(uint) {}
 
-function f4(int) public {}
-function f4(int, int) public {}
+function f4(int) {}
+function f4(int, int) {}
 
-function f5(int) public {} //~ ERROR: function with same name and parameter types declared twice
-function f5(int) public {}
+function f5(int) {} //~ ERROR: function with same name and parameter types declared twice
+function f5(int) {}
 
-function f6(string memory) public {} //~ ERROR: function with same name and parameter types declared twice
-function f6(string calldata) public {}
+function f6(string memory) {} //~ ERROR: function with same name and parameter types declared twice
+function f6(string calldata) {}
 
-function f7(string storage) internal {}
-function f7(string memory) public {}
+// function f7(string storage) internal {} 
+// function f7(string memory) {}
```

Comment:
@DaniPopes If I don't comment this out, it's causing many errors in this file to be skipped (not emitted). Please can you help me understand what's happening? I see that `f8` functions below have also been commented probably for similar reason

Reply:
Compilation fails at an earlier stage, so the later stages which emit these errors don't get reached; commenting out is fine